vendor/bat-native-ads/src/bat/ads/internal/account/confirmations/confirmation_payload_json_writer.h

```diff
@@ -1,17 +1,22 @@
-/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
  * This Source Code Form is subject to the terms of the Mozilla Public
  * License, v. 2.0. If a copy of the MPL was not distributed with this file,
  * You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-#ifndef BRAVE_VENDOR_BAT_NATIVE_ADS_SRC_BAT_ADS_INTERNAL_ACCOUNT_CONFIRMATIONS_CONFIRMATIONS_UTIL_H_
-#define BRAVE_VENDOR_BAT_NATIVE_ADS_SRC_BAT_ADS_INTERNAL_ACCOUNT_CONFIRMATIONS_CONFIRMATIONS_UTIL_H_
+#ifndef BRAVE_VENDOR_BAT_NATIVE_ADS_SRC_BAT_ADS_INTERNAL_ACCOUNT_CONFIRMATIONS_CONFIRMATION_PAYLOAD_JSON_WRITER_H_
+#define BRAVE_VENDOR_BAT_NATIVE_ADS_SRC_BAT_ADS_INTERNAL_ACCOUNT_CONFIRMATIONS_CONFIRMATION_PAYLOAD_JSON_WRITER_H_
+
+#include <string>
 
 namespace ads {
 
 struct ConfirmationInfo;
 
-bool VerifyConfirmation(const ConfirmationInfo& confirmation);
+namespace JSONWriter {
```

Comment:
Namespace should be lower-case with words separated by underscores

Reply:
This one is consistent with other usages in ads. There will be a follow-up issue to remove this namespace throughout ads.

Reply:
Maybe we can do a class with static function instead of this namespace?

Reply:
Sounds good but would prefer in a follow up PR which will land tomorrow

Reply:
Fixed